sig-and-wg/sig-core/proposals/event-bus-monitoring-metrics.md

```diff
@@ -0,0 +1,113 @@
+# Event Bus Monitoring Metrics Proposal
+
+## Event Bus
+- **Throughput**
```

Comment:
Please assess if the metrics such as throughput are able to serve troubleshooting purpose with event delivery. Please add some details about how they can help. In case they are not in the scope of this overview, please remove them. In general, this comment applies to all the metrics

Reply:
I wouldn't go for removing the performance metrics as long as we are looking into system health monitoring generally. Imagine the operation is troubleshooting why lambda is not delivered for a few minutes, this might be because of a problem/miss-config or just because of the latency in the queue.
Performance metrics are important in the system troubleshooting generally and specifically in some cases.
I'd also suggest prioritizing the implementation of the proposed metrics rather than removing them.

Reply:
Again, please provide details of why and how each metric can support troubleshooting.

Reply:
Since this is a proposal, we need to have proper reasoning `why` such and such metric would make sense. The intent of this task is to aid the planning/refinement for the task to do the actual implementation. Just having a plain list is not sufficient for such a purpose.

Reply:
Interesting is that you are already providing a reasonable description in the comments. It would be nice to put this in the document.